plugins/armorblox/icon_armorblox/triggers/get_incidents/schema.py

```diff
@@ -0,0 +1,329 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Component:
+    DESCRIPTION = "Get a list of incidents identified by Armorblox, by default it starts polling all the incidents since last day"
```

Comment:
```suggestion
    DESCRIPTION = "Get a list of incidents identified by Armorblox. By default, it starts querying for all the incidents since the previous day"
```